query-node/mappings/package.json

```diff
@@ -16,14 +16,15 @@
     "bootstrap-data:fetch": "yarn bootstrap-data:fetch:members && yarn bootstrap-data:fetch:workingGroups && yarn bootstrap-data:fetch:categories"
   },
   "dependencies": {
-    "@polkadot/types": "8.9.1",
+    "@apollo/client": "^3.2.5",
     "@joystream/hydra-common": "5.0.0-alpha.4",
     "@joystream/hydra-db-utils": "5.0.0-alpha.4",
-    "@joystream/warthog": "^2.41.9",
     "@joystream/js": "^1.4.0",
-    "@apollo/client": "^3.2.5"
+    "@joystream/warthog": "^2.41.9",
+    "@polkadot/types": "8.9.1"
   },
   "devDependencies": {
+    "eslint-plugin-local-rules": "2.0.0",
```

Comment:
Adding this package seems to require native build on mac/arm64 so I had to add `node-gyp` as a dev dependency to successfully build docker image of query-node.

Reply:
Addressed in [810240d](https://github.com/Joystream/joystream/pull/4856/commits/810240d2ae555d114778650db8b76823769a603e)